src/test/java/org/swisspush/redisques/RedisQuesTest.java

```diff
@@ -247,6 +248,7 @@ public void getQueuesFiltered(TestContext context) {
                 context.assertEquals(OK, message.result().body().getString(STATUS));
                 asyncEnqueue.countDown();
             });
+            Thread.sleep(10);
```

Comment:
Why is this necessary? The test will wait until the countDown has gone from 20 to 0